<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>430</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QgsFieldComboBox" name="Field2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>160</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QgsFieldComboBox" name="Field1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>160</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="legendCheck">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Legend</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QgsMapLayerComboBox" name="LayerCombo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>160</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="filePath">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>360</y>
     <width>371</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="browseButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>360</y>
     <width>26</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFieldComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsfieldcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>LayerCombo</sender>
   <signal>layerChanged(QgsMapLayer*)</signal>
   <receiver>Field1</receiver>
   <slot>setLayer(QgsMapLayer*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LayerCombo</sender>
   <signal>layerChanged(QgsMapLayer*)</signal>
   <receiver>Field2</receiver>
   <slot>setLayer(QgsMapLayer*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
